proto/provenance/attribute/v1/tx.proto

```diff
@@ -42,6 +42,9 @@ message MsgAddAttributeRequest {
   string account = 4;
   // The address that the name must resolve to.
   string owner = 5;
+  // expiration is an optional expiration timespan string, e.g. "1y" (see expiration module for format)
```

Comment:
You have this in another comment: ```  // the accepted values for a duration are n{h,d,w,y}
  // where 1h = 60m, 1d = 24h, 1w = 7d (or 168h), 1y = 365d (or 8760h) ```
Maybe add that instead?

Reply:
updated comments.

>// expiration is an optional expiration timespan string, e.g. "1y"
  // If provided, it sets an expiration with the given time period for the attribute.
  // accepted values are n{h,d,w,y}
  // where 1h = 60m, 1d = 24h, 1w = 7d (or 168h), 1y = 365d (or 8760h)